backend/pyproject.toml

```diff
@@ -27,6 +27,12 @@ shortuuid = "^1.0.13"
 pytz = "^2024.1"
```

Comment:
you will need to update the poetry version to 3.12 as well

Reply:
What do you mean by saying to update the poetry version?